ingestion/src/metadata/ingestion/connections/builders.py

```diff
@@ -117,6 +121,30 @@ def init_empty_connection_options() -> ConnectionOptions:
     return ConnectionOptions(__root__={})
 
 
+def _add_password(url, connection, attr) -> str:
+    # A helper function that adds the password to the url if it exists
+    password = getattr(connection, attr, None)
```

Comment:
would be good to have some hints on the arguments. Also the name `attr` is not quite clear on what we want to do

Reply:
this could use a docstring as well

Reply:
makes sense, let me make it more readable.